stubs/setuptools/METADATA.toml

```
@@ -1,4 +1,4 @@
-version = "75.8.*"
+version = "~=75.8.2"
```

```suggestion
version = "75.8.2"
```
Let's maybe just pin exactly. setuptools doesn't do fourth digit versions / in practice I find `~=` semantics a little confusing

---

This isn't so much about a fourth digit. `~=7.8.2` is basically equivalent to `7.8.*`, but where `* >= 2`.

---

Okay I guess at least what I said about always confusing myself about `~=` semantics is true :-)